priv/static/assets/demo/default/custom/components/datatables/base/sites_local_datatable.js

```diff
@@ -0,0 +1,394 @@
+var sitesDataTable = null;
+
+var DatatableDataSites = function() {
+  sitesDataTable = $(".m_commands_datatable").mDatatable({
+    data: {
+      type: "remote",
+      speedLoad: true,
+      source: "/get_all_sites",
+      pageSize: 50,
+      serverPaging: false,
+      serverFiltering: false,
+      serverSorting: false
+    },
+    layout: {
+      theme: "default",
+      class: "",
+      scroll: !1,
+      height: 950,
+      footer: !1
+    },
+    sortable: !0,
+    filterable: !1,
+    pagination: false,
+    columns: [
+    {
+      field: "actions",
+      width: 60,
+      title: "Actions",
+      locked: {left: 'xl'},
+      sortable: false,
+      template: function(t) {
+        return "<div class='editRULE cursor_to_pointer fa fa-edit' data-id='"+ t.id +"'></div> <div class='deleteSite cursor_to_pointer fa fa-trash' data-id='"+ t.id +"'></div>";
```

Comment:
Edit Rule should be Edit site. and also change it to the jquery method as well. where you are calling the edit.